src/Look/Look.cpp

```diff
@@ -49,7 +49,8 @@ Look::InitialiseConfigured(const UISettings &settings,
   trace_history.Initialise(settings.info_boxes.inverse);
   info_box.Initialise(settings.info_boxes.inverse,
                       settings.info_boxes.use_colors,
-                      infobox_width);
+                      infobox_width,
+		      settings.info_boxes.scale_title_font);
```

Comment:
Wrong indent

Reply:
I'm using this in my .vimrc
```
set sw=2
set noexpandtab
```
Is that not OK?

Reply:
I don't care how you configure the editor of your choice or which editor you use. What matters is whether your code is properly formatted. And obviously, it's not.